config/routes_into_teaching.yml

```
@@ -28,6 +28,15 @@ routes:
         answers: ["*"]
       - question: location
         answers: ["*"]
+    highlight:
+      text: "Most popular route"
+      matches:
+        - question: undergraduate_degree
+          answers: ["Yes"]
```

```suggestion
          answers: ["Yes", "Not yet"]
```
@spencerldixon I presume we can make this change so the route is also highlighted for people currently studying for a degree?

NB this isn't test data - this is how we want it to work!